src/SDK/Trace/Behavior/HttpSpanExporterTrait.php

```diff
@@ -46,14 +45,20 @@ protected function doExport(iterable $spans): int
         try {
             $response = $this->dispatchSpans($spans);
         } catch (ClientExceptionInterface $e) {
+            $this->error('Error exporting span', ['error' => $e->getMessage()]);
+
             return $e instanceof RequestExceptionInterface
                 ? SpanExporterInterface::STATUS_FAILED_NOT_RETRYABLE
                 : SpanExporterInterface::STATUS_FAILED_RETRYABLE;
         } catch (Throwable $e) {
+            $this->error('Error exporting span', ['error' => $e->getMessage()]);
```

Comment:
I don't think `STATUS_FAILED_NOT_RETRYABLE` and `STATUS_FAILED_RETRYABLE`should have the same severity level.